src/Solver/Interoperability.cpp

```diff
@@ -551,8 +647,55 @@ void seissol::Interoperability::setMaterial(int i_meshId, int i_side, double* i_
     assert(side < 4);
     material = &m_ltsLut.lookup(m_lts->material, i_meshId - 1).neighbor[side];
   }
+  //TODO: Lukas fix this ;-)
```

Comment:
Could you change this to TODO(Lukas)?
Otherwise I won't find it by grepping

Reply:
Oh, and maybe a comment would be nice, such as:
// Use placement new because pointer to virt. function table gets overwritten by 0 during init.